l2geth/consensus/clique/clique.go

```diff
@@ -324,10 +324,16 @@ func (c *Clique) verifyCascadingFields(chain consensus.ChainReader, header *type
 	if parent == nil || parent.Number.Uint64() != number-1 || parent.Hash() != header.ParentHash {
 		return consensus.ErrUnknownAncestor
 	}
-	// [REMOVED] to account for timestamp changes
-	//if parent.Time+c.config.Period > header.Time {
-	//	return ErrInvalidTimestamp
-	//}
+
```

Comment:
Instead of commenting out the code, place it behind the `UsingOVM` flag and add a comment as to why the diff exists